example/assets/preview/config.yml

```diff
@@ -0,0 +1,166 @@
+backend:
+  name: github
+  repo: netlify/netlify-cms
```

Comment:
If we use the CMS repo on the GitHub backend for deploy previews, contributors that do not have write permissions will not be able to use our deploy previews. Right?